pkg/authorization/api/validation/validation.go

```diff
@@ -222,6 +225,52 @@ func ValidateRoleBinding(roleBinding *authorizationapi.RoleBinding, isNamespaced
 		}
 	}
 
+	for i, subject := range roleBinding.Subjects {
+		subjectLocation := fmt.Sprintf("subjects[%d]", i)
```

Comment:
It would have been better to break this into a function ValidateRoleBindingSubject so that you could use Prefix().